test/runtests.jl

```diff
@@ -8,7 +8,7 @@ using JSON
 dataset_list = get(DatasetListRequest(limit = 4))
 @test length(dataset_list) == 4
 
-info = get(DatasetMetadataRequest(id = "5627f42b9fc3c132be08d84f"))
+#info = get(DatasetMetadataRequest(id = "5627f42b9fc3c132be08d84f"))
```

Comment:
why ?

Reply:
Got this response, so I just chose the first id from the preceding ```DatasetListRequest```:
```
julia> info = get(DatasetMetadataRequest(id = "5627f42b9fc3c132be08d84f"))
ERROR: HTTP.ExceptionRequest.StatusError(400, HTTP.Messages.Response:
"""
HTTP/1.1 400 Bad Request
Server: nginx/1.10.3 (Ubuntu)
Date: Wed, 06 Feb 2019 13:50:40 GMT
Content-Type: application/json
Content-Length: 77
Connection: keep-alive
Allow: DELETE, GET, HEAD, OPTIONS, PATCH, POST, PUT

{"message": "Invalid dataset id (5627f42b9fc3c132be08d84f).", "type": "rest"}""")
```

Reply:
oh, so I guess this is just terribly out of date